modules/microvm/virtualization/microvm/guivm.nix

```diff
@@ -106,6 +110,7 @@ let
             # Logging client configuration
             logging.client.enable = config.ghaf.logging.client.enable;
             logging.client.endpoint = config.ghaf.logging.client.endpoint;
+            logging.identifierFilePath = "/tmp/MACAddress";
```

Comment:
This is probably needed for the control panel, which uses the hardcoded /tmp/MACAddress. It might be worth adding a comment here for the future in case getting the MAC address is implemented in GIVC.

Reply:
The commit is added.